components/menu/Navigation.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from "react";
+import { MenuItem, MenuItemMobile } from "./MenuItem";
+import { useRouter } from "next/router";
+
+const Navigation: FC<{
+  navigation: {};
```

Comment:
Is there a better type for this?

Reply:
Oops, yeah that was meant as a placeholder.  I have made an update in the latest push.